packages/odyssey/src/scss/vendors-ext/_choices-ext.scss

```diff
@@ -0,0 +1,453 @@
+    background-size: 8px;
+
+    &:hover,
+    &:focus {
+      opacity: 1;
+    }
+
+    &:focus {
+      box-shadow: 0 0 0 2px cv('action', 'base');
+    }
+  }
+
+  .ods-select--item[data-value=''] .ods-select--button {
+    display: none;
+  }
+
+  &::after {
+    content: '';
+    position: absolute;
+    top: 50%;
+    right: $em-small-spacing;
+    width: ms(-1);
+    height: ms(-1);
+    transform: translateY(-50%);
+    background-image: get-icon('caret', $text-body);
+    background-repeat: no-repeat;
+    background-size: ms(-1);
+    pointer-events: none;
+  }
+
+  &.is-ods-select-open {
+    &::after {
+      transform: translateY(-50%) rotate(180deg);
+    }
+  }
+
+  &[dir='rtl'] {
+    &::after {
+      right: auto;
+      left: 11.5px;
+    }
+
+    .ods-select--button {
+      right: auto;
+      left: 0;
+      margin-right: 0;
+      margin-left: 25px;
+    }
+  }
+}
+
+.ods-select[data-type*='select-multiple'],
+.ods-select[data-type*='text'] {
+  .ods-select--inner {
+    cursor: text;
+  }
+
+  .ods-select--button {
+    display: inline-block;
+    position: relative;
+    width: $choices-button-dimension;
+    margin-top: 0;
+    margin-right: -$choices-button-offset/2;
+    margin-bottom: 0;
+    margin-left: $choices-button-offset;
+    padding-left: $choices-button-offset * 2;
+    border-left: 1px solid darken(cv('action', 'base'), 10%);
+    border-radius: 0;
+    opacity: 0.75;
+    background-image: $choices-icon-cross;
+    background-size: $choices-button-dimension;
+    line-height: 1;
+
+    &:hover,
+    &:focus {
+      opacity: 1;
+    }
+  }
+}
+
+.ods-select--inner {
+  display: inline-block;
+  width: 100%;
+  padding: $em-tiny-spacing $em-small-spacing;
+  overflow: hidden;
+  transition-property: border-color, background-color, box-shadow;
+  transition-duration: $base-duration;
+  transition-timing-function: $base-timing;
+  border: 1px solid $base-border-color;
+  border-radius: $base-border-radius;
+  background-color: $white;
+  box-shadow: 0 0 0 0 fauxpacity(cv('action', 'base'));
+  color: $text-body;
+  font-family: $body-font-family;
+  font-size: ms(0);
+  line-height: $base-line-height;
+  vertical-align: top;
+
+  &:hover,
+  .is-ods-select-focused &,
+  .is-ods-select-open & {
+    border-color: cv('action', 'base');
+  }
+
+  .is-ods-select-open &,
+  [data-invalid] .is-ods-select-open & {
+    border-radius: $base-border-radius $base-border-radius 0 0;
+    border-bottom-color: $base-border-color;
+  }
+
+  .is-ods-select-flipped.is-ods-select-open &,
+  [data-invalid] .is-ods-select-flipped.is-ods-select-open & {
+    border-radius: 0 0 $base-border-radius $base-border-radius;
+    border-top-color: $base-border-color;
+    border-bottom-color: cv('action', 'base');
+  }
+}
+
+.ods-select--list {
+  margin: 0;
+  padding-left: 0;
+  list-style: none;
+}
+
+.ods-select--list--single {
+  display: inline-block;
+  width: 100%;
+
+  [dir='rtl'] & {
+    //padding-right: 4px;
```

Comment:
Remove?

Reply:
Keeping as a reminder for when we turn on more features and start worrying about RTL support.